src/store/index.js

```diff
@@ -0,0 +1,233 @@
+import { applyMiddleware, createStore } from 'redux';
+import thunk from 'redux-thunk';
+
+const ACTION_TYPES = {
+  ADD_POSTS: 'POSTS::ADD',
+  SWITCH_LOADING: 'LOADING::SWITCH',
+  IS_INITIALIZED: 'INITIALIZED::IS',
+  HAS_ERROR: 'ERROR::HAS',
+  IS_SORT: 'IS::SORT',
+  ADD_SORTED_POSTS: 'SORTED_POSTS::ADD',
+  DELETE_POST: 'POST::DELETE',
+  ADD_TEXT: 'TEXT::ADD',
+  POSTS_SORT: 'POST::SORT',
+  RESET_FILTER: 'RESET::FILTER',
+  DELETE_COMMENT: 'DELETE::COMMENT',
```

Comment:
1. action can't have names like `IS_INITIALIZED` or `IS_SORT` etc., these names are used to mark some state,
2. if you write name, you need to specify the main entity first and then an action with it, i.e. `COMMENT::DELETE`
3. are you sure you need so many action types?

Reply:
3. I need all this action types